querybook/webapp/hooks/dag/useSavedDAG.ts

```diff
@@ -30,7 +30,12 @@ export function useSavedDAG(docId: number) {
     ]);
 
     const onSave = React.useCallback(
-        (nodes: Node[], edges: Edge[], meta?: Record<string, any>) => {
+        (
+            nodes: Node[],
+            edges: Edge[],
+            exporterMeta?: Record<string, any>,
+            useTemplatedVariables?: boolean
```

Comment:
need more generic way to handle `useTemplatedVariables`, imagine it would be an dictionary of values